src/html/HTMLWriter.jl

```diff
@@ -807,9 +807,8 @@ function render(doc::Documenter.Document, settings::HTML=HTML())
     all(size_limit_successes) || throw(HTMLSizeThresholdError())
 
     open(joinpath(doc.user.build, ctx.search_index_js), "w") do io
-        println(io, "var documenterSearchIndex = {\"docs\":")
         # convert Vector{SearchRecord} to a JSON string + do additional JS escaping
-        println(io, JSDependencies.json_jsescape(ctx.search_index), "\n}")
+        println(io, JSDependencies.json_jsescape(ctx.search_index))
```

Comment:
Needs to be double checked, but JSON might need a different escaping function.

Reply:
It looks to me like json_jsescape only escapes some newline unicode, so it turns already valid JSON to also be valid javascript (also, according to comments in the source code, this is not required since 2019).

More closely looking at the function, it calls `JSON.json()`, and then escapes the problematic unicode in whatever is returned, so this should be fine (at least for this change).